R/Functions_L0_auxilary.R

```diff
@@ -1280,8 +1276,6 @@ getCasteId <- function(x, caste = "all", collapse = FALSE, simParamBee = NULL) {
 #'   vector with sex information
 #' @param simParamBee \code{\link[SIMplyBee]{SimParamBee}}, global simulation parameters
 #'
-#' @seealso \code{\link[SIMplyBee]{getCaste}}
```

Comment:
@gregorgorjanc , did you get rid of this one on purpose or by msitake?

Reply:
There is another one few lines below, so I removed a duplicate